From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sunu Aziz Rahayu <sunuazizrahayu@users.noreply.github.com>
Date: Fri, 26 Sep 2026 00:00:00 +0700
Subject: [PATCH] base: drop product_charger_res_images to fix duplicate

Modul Soong product_charger_res_images menginstal
product/etc/res/images/charger/battery_fail.png, bertabrakan dengan
modul charger milik device/phh/treble sehingga kati gagal:

  overriding commands for target
  `.../system/product/etc/res/images/charger/battery_fail.png'

Charger bawaan treble sudah cukup untuk GSI, buang yang AOSP.

---
 base.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/base.mk b/base.mk
index 0000000..0000000 100644
--- a/base.mk
+++ b/base.mk
@@ -50,10 +50,6 @@ PRODUCT_COPY_FILES += \
 PRODUCT_COPY_FILES += \
     device/phh/treble/charger.rc:system/etc/init/charger.rc

-# Charger image
-PRODUCT_PACKAGES += \
-    product_charger_res_images
-
 # USB Audio
 PRODUCT_COPY_FILES += \
     frameworks/av/services/audiopolicy/config/usb_audio_policy_configuration.xml:system/etc/usb_audio_policy_configuration.xml \
--
2.54.0
